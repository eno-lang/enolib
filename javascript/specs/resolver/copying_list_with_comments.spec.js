const { parse } = require('../..');

const input = `
languages:
> eno-lang.org
- eno
> json.org
- json

copy < languages
> yaml.org
- yaml
`.trim();

describe('Resolution', () => {
  describe('Copying list with comments', () => {
    it('works', () => {
      const doc = parse(input);
      expect(doc.raw()).toMatchSnapshot();
    });
  });
});
