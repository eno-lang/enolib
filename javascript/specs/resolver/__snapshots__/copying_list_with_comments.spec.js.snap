// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resolution Copying list with comments works 1`] = `
Array [
  Object {
    "languages": Array [
      "eno",
      "json",
    ],
  },
  Object {
    "copy": Array [
      "eno",
      "json",
      "yaml",
    ],
  },
]
`;
